lib/services/project-changes-service.ts

```diff
@@ -227,11 +232,8 @@ export class ProjectChangesService implements IProjectChangesService {
 
 			let fileStats = this.$fs.getFsStats(filePath);
 
-			let changed = fileStats.mtime.getTime() >= this._outputProjectMtime || fileStats.ctime.getTime() >= this._outputProjectCTime;
-			if (!changed) {
-				let lFileStats = this.$fs.getLsStats(filePath);
-				changed = lFileStats.mtime.getTime() >= this._outputProjectMtime || lFileStats.ctime.getTime() >= this._outputProjectCTime;
-			}
+			let changed =  fileStats.mtime.getTime() > this._outputProjectMtime ||
```

Comment:
why is this without `=` sign, but the next check has it? I believe you need the `=` in both cases, as the `_outputProjectMtime ` is set after the prepare. In case it is equal with a file's mtime, this means the file has been modified when the prepare is finishing (exactly at the same time), so we need to prepare the project again.